rapids-cmake/cython/create_modules.cmake

```
@@ -61,7 +61,8 @@ $ORIGIN.
   A list of targets that are associated with the Cython targets created in this
   function. The target<-->associated target mapping is stored and may be
   leveraged by the following functions:
-  - :code:command:`rapids_cython_add_rpath_entries` accepts a path for an
+
+  - :code:`rapids_cython_add_rpath_entries` accepts a path for an
```

I think this was supposed to be `:cmake:command:` instead of `:code:command:`.
```suggestion
  - :cmake:command:`rapids_cython_add_rpath_entries` accepts a path for an
```